lib/rules.ex

```diff
@@ -19,8 +19,8 @@ defmodule Request.Validator.Rules do
   @spec bail(list(atom() | tuple())) :: Request.Validator.Rules.Bail.t()
   def bail(rules), do: %__MODULE__.Bail{rules: rules}
 
-  @spec map(maybe_improper_list()) :: Request.Validator.Rules.Map_.t()
-  def map(attrs), do: %__MODULE__.Map_{attrs: attrs}
+  @spec map(maybe_improper_list(), boolean()) :: Request.Validator.Rules.Map_.t()
+  def map(attrs, nullable \\ false), do: %__MODULE__.Map_{attrs: attrs, nullable: nullable}
```

Comment:
adding a new function `nullable_map` makes the API cleaner. Also, where is the implementation where this use case is handled? 🤔

Reply:
> adding a new function nullable_map makes the API cleaner

I thought so too, until I found out `map()` already has the `nullable` field. 

What's the purpose of adding a new function when the current function supports null values?

Reply:
`nullable_map` is explicit as opposed to `map(attrs, true)`. I will take a look at this functionality today.

Reply:
They're both explicit, one in the name and the other in the arguments.

I don't think one form is better than the other.

Reply:
see https://github.com/heywhy/elixir-request-validator/blob/ff23eacb2f2b4954d2df2ffbf6561aa4bb1bd980/lib/helper.ex#L96, it's a functionality that exists already.